github/Organization.pyi

```diff
@@ -66,6 +66,13 @@ class Organization(CompletableGithubObject):
         allow_merge_commit: Union[bool, _NotSetType] = ...,
         allow_rebase_merge: Union[bool, _NotSetType] = ...,
     ) -> Repository: ...
+    def create_secret(
+        self,
+        secret_name: str,
+        unencrypted_value: str,
+        visibility: str,
+        selected_repositories: List[Repository],
```

Comment:
I think you need to mark the visibility parameter with it's default value. Also, what does type checking return if you call create_secret() with no selected_repositories?

Reply:
Marked and set NotSet as default value for selected_repositories